proco/connection_statistics/tasks.py

```diff
@@ -43,11 +48,19 @@ def load_data_from_unicef_db(*args):
 
 @app.task
 def update_real_time_data():
-    chain(
-        # it would be better to use group, but we need result backend to be configured
-        # group(load_data_from_unicef_db.s(), load_brasil_daily_statistics.s()),
+    countries_aggregate_daily_statistics_tasks_chain = [
+        aggregate_daily_statistics.s(country_id)
+        for country_id in Country.objects.values_list('id', flat=True)
+    ]
+    tasks_chain = [
         load_data_from_unicef_db.s(),
         load_brasil_daily_statistics.s(),
         aggregate_real_time_data.s(),
-        aggregate_daily_statistics.s(),
+    ] + countries_aggregate_daily_statistics_tasks_chain + [
```

Comment:
please replace aggregate_daily_statistics chaining with group to allow them being executed in parallel.
also it would be good to split update_countries_weekly_statuses_task in similar way